### DISCUSSION TITLE
Full TA/RA Simulation

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#623] [Ideas]
Will say traffic traffic etc (the TA), then climb or descend etc(RA). like in the real plane - would be very very cool when near to AI (like livetraffic or traffic global) and multiplayer (vatsim/Ivao) traffic! 
it’s already in the zibo and toliss and others, you could use for reference.

**Comment by aeroniemi** [comment]
libtcas exists and could probably be integrated, would require some modifications for the 787 though
also need more sound files as the current soundpack set are unfinished

**Reply by Simleybob** [reply]
Sounds good!
